### ISSUE TITLE
Would it possible to support multiple views inputs?

### ISSUE THREAD

**Issue by hp-cv** [liuyuan-pal/SyncDreamer#26]
Hi, the single shot image input provides reasonably good result.
I wonder if there's a way to support multiple views(at least 2 or 3+ views,could be under the same angles) that can increase the quality?Thanks.

**Comment by clinplayer** [comment]
Yes! If you know the camera poses, a potential way is to unproject the multiview image features to the 3D feature volume, and then you can condition the denoising process on the correlated multiview features. To achieve this, you may need to finetune the network using a multiview setting.
